espresso-server/app/build.gradle.kts

```diff
@@ -101,6 +116,11 @@ dependencies {
     androidTestImplementation("org.nanohttpd:nanohttpd-webserver:${Version.nanohttpd}")
     androidTestImplementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:${Version.kotlin}")
     androidTestImplementation("org.jetbrains.kotlin:kotlin-reflect:${Version.kotlin}")
+    androidTestImplementation("androidx.compose.ui:ui-test:${Version.compose}")
+    androidTestImplementation("androidx.compose.ui:ui-test-junit4:${Version.compose}"){
```

Comment:
can we use junit5 or we are limited here?

Reply:
There is no junit5 available in mvn repo